src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx

```diff
@@ -44,6 +54,44 @@ const ManageMemberModal: FC<ManageMemberModalProps> = ({
         placeholderText={{ id: 'members.modal.selectActions' }}
         isListRelative
```

Comment:
What does ' isListRelative' prop mean? I haven't seen it earlier. I think this name isn't correct.

Reply:
This prop changes position of list in select to relative from absolute.